ChangeLog

```diff
@@ -1,5 +1,61 @@
 OpenContainers Specifications
 
+Changes with v1.0.2:
+
+	Additions:
+
+	* Add create-container, create-runtime and start-container hooks (#1008)
+	* config-linux: add Intel RDT CLOS name sharing support (#988)
+	* config-linux: Add Intel RDT/MBA Linux support (#932)
+	* config-linux: Add Memory cgroup's use_hierarchy (#985)
+	* Add Linux personality support (#1012)
+	* config: Add Windows Devices to Schema (#976)
+	* Add support for SCMP_ACT_LOG (#1019)
+	* config-linux: support seccomp flags (#1018)
+
+	Minor fixes and documentation:
+
+	* Makefile: avoid SELinux for making docs
```

Comment:
If I'm not mistaken, you haven't picked up the changes that happened since #1026 in the log. Here's a list in chronological order of merges, with slight manual editing.

```suggestion
	* Makefile: avoid SELinux for making docs
	* Clarify case with pre-configured Intel RDT closID (#1034)
	* config-linux: describe more about rootfs mount propagation (#1035)
	* config-linux: add SHOULD to linux.namespaces.type (#1025)
	* Reduce DCO checks per PR from 3 to 1 (#1029)
	* Fix typo in RELEASES.md (#1033)
	* Remove some unneeded indent (#1031)
	* Add documentation how to do releases (#1027)
	* Removed Vishnu Kannan & Brandon Philips from maintainers (#1030 & #1028)
```

Reply:
oh you're right. I forgot to update the ChangeLog.
I wish automate that step. It is far too manual currently.